intro_to_R_tidyverse/intro_to_tidyverse.Rmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "Intro to Tidyverse"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2019**
+
+In this notebook, we will take you through the basics of the tidyverse: 
+a collection of packages that make R use easier and more convenient.  
+
+There are a lot of tutorials and free books Hadley Wickham and friends have on 
+how to use the [tidyverse packages](https://r4ds.had.co.nz/index.html). 
+[Datacamp](https://www.datacamp.com/) also has a bunch of good tutorials on this. 
+
+## Set Up
+R packages are collections of functions. 
+Installing R on your computer installs `base` functions like statistical tests 
+and some plotting functionality.
+The `tidyverse` is a collection of packages that are handy for general data 
+wrangling, analysis, and visualization. 
+Other packages that are specifically handy for different biological analyses are 
+found on [Bioconductor](https://www.bioconductor.org/).
+If we want to use a package's functions we first need to install them.
+In our Docker container, we already have `tidyverse` and other packages we will
+use in this workshop installed for you. 
+But if you needed to install it or other packages available on CRAN, you 
+do it using the `install.packages` function like this: `install.packages("tidyverse")`
+```{r Load tidyverse}
+library(tidyverse)
+```
+
+"metadata.tsv" is a file we will use for practicing the tidyverse.
+This metadata is from GSE30074 [refine.bio](https://www.refine.bio), a gene 
+expression dataset from medulloblastoma.
+  
+First let's read in the dataset using the tidyverse package: `readr`.
+Although base R has read-in functions, the `readr` functions are faster and more
+straight forward to use so we are going to use those here. 
+Because the file we are reading in is a .tsv file we will be using the 
+`read_tsv` function. 
+But note that there are analogous functions for .csv files (`read_csv`) and 
+other files types.
+
+### About directories
+Before we can import the file we need though, we should double check what
+directory R is in, aka what our current *directory* is. 
+We can do this by using the `getwd()` function, which will tell us what folder
+we are in. 
+Then we can use the `dir()` function to have R list the files in our current 
+directory. 
+What we should see is the file `metadata.tsv` show up as one of the files when 
+we run `dir()`.
+If for some reason we were not in the correct directory, we can either set our
+directory to somewhere else using `setwd()` OR we can tell R to look for the 
+file `metadata.tsv` somewhere else using a relative `file.path()`.
+We will discuss and practice these concepts more in other modules as we go along.
```

Comment:
I don't believe we explicitly talk about relative vs. absolute paths anywhere else (and even if we do, repetition is helpful), so I think this should be a bit more in depth here.

Reply:
Alternatively, include as part of intro (#25)

Reply:
I think this section is a good a time as any for them to practice it, but we might wanna introduce the concept in the slides (#25).

Reply:
I'll add more details on this section though too.

Reply:
Per our in person convoy: I'm gonna hold off on expanding this and other sections much more before we get what we have right now pushed.